xbmc/FileItem.cpp

```diff
@@ -1184,6 +1184,12 @@ bool CFileItem::IsNFO() const
   return URIUtils::HasExtension(m_strPath, ".nfo");
 }
 
+bool CFileItem::IsVideoExtras() const
+{
+  return m_bIsFolder &&
+         StringUtils::EqualsNoCase(URIUtils::GetFileOrFolderName(m_strPath), "extras");
```

Comment:
feature request for a follow up PR: make the name configurable via `advancedsettings.xml` and have it check for `extras|specials` by default. (note: new advanced setting needs to be added to wiki, not necessarily by you, just to not forget to ping wiki people)

Reply:
Hi @da-anda , I tend to implement this as a setting like "Tracking name template" setting for music, use "|" as a separator for different folder names, and initial value is "extras|specials|featurettes". User can edit this easier than advancedsettings.xml.

Reply:
we don't want to clutter the UI with settings that most people won't change or are heavily technical, so advancedsettings.xml is IMHO the way to go here. Also, this would be in line with all the other scraping related settings (`excludefromscan`, `discstubextensions`, `videoextensions`, ...)

Reply:
I see. I'll follow this.

Reply:
honestly no idea why `media -> music -> thumbnail image files` or `media -> music -> track naming template` are available in settings. I recall that at some point (years ago) it was discussed to move some of the advancedsettings into the UI, but IIRC that never happened and I don't recall the consensus from back then. @jjd-uk do you remember by any chance?
I am not in general against moving the more common advanced settings to UI settings (in expert level or even a new level), but we need documented guidelines for this first

Reply:
@xodidox please wait on additional feedback before implementing this, to not waste your time on the wrong setting type

Reply:
sorry, not really. As everybody is quite busy this time of the year, not many attended the last team meeting, so technical things had not been discussed. I'd say go ahead with what you have in mind, but if you make it a setting, it should be `advanced`  or even `expert`  level I think

Reply:
I see, then I'll follow the music track template and implement it in that way. Keep it as expert level.